### PR TITLE
Tweak Wharf

### PR DESCRIPTION
**Pull request by mfalesni**
Wharf is now living inside the webdriver. Plus I have removed the `%s`'s

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3915712684d46d709c667bad4a2cfd3b0e5bfdb3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/browser.py [hunk 1]
```diff
@@ -139,12 +146,12 @@ def start(webdriver_name=None, base_url=None, **kwargs):
                 co['args'].append(arg)
             browser_kwargs['desired_capabilities']['chromeOptions'] = co
 
-    if thread_locals.wharf:
+    if wharf is not None:
```
Comment (psav) [inline utils/browser.py]: Can a Wharf object be equivalent to None?

Comment (mfalesni) [inline utils/browser.py]: Yes, if there was no wharf before and if wharf is not set up in env.yaml.

Comment (psav) [inline utils/browser.py]: OK can we put a comment just to state why the ```is not None``` is there

**Comment by dajoRH** [comment]
Lint report for commit 30f7218e4867b226fb873ad44bfd179a0ef6253a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 30f7218e4867b226fb873ad44bfd179a0ef6253a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 30f7218e4867b226fb873ad44bfd179a0ef6253a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
@mfalesni if this works for smoke we think it will be OK elsewhere?

**Comment by mfalesni** [comment]
@psav still not sure if I want this change or not ...

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by RonnyPfannschmidt** [comment]
that one conflicts with #3182

**Comment by psav** [comment]
Shall we close this off now that the new browser manager is in the works?

**Comment by mfalesni** [comment]
Yes

**Pull request closed without merging**